launcher/package/rpm/MultiMC5.spec

```diff
@@ -1,14 +1,20 @@
 Name:           MultiMC5
 Version:        1.4
-Release:        3%{?dist}
+Release:        4%{?dist}
 Summary:        A local install wrapper for MultiMC
 
 License:        ASL 2.0
 URL:            https://multimc.org
-BuildArch:      x86_64
+ExclusiveArch:  %{ix86} x86_64
```

Comment:
`%{ix86}` includes all of `i386 i486 i586 i686 pentium3 pentium4 athlon geode`
I think these should work with the 32 bit tarball, but correct me if I am wrong